docs/Contributing.asciidoc

```diff
@@ -211,7 +211,7 @@ it when running tests. Example:
 
 [source,sh]
 ----
-TEST_PG='DBI:Pg:dbname=openqa_test;host=/dev/shm/tpg' OPENQA_BASEDIR= prove -v t/14-grutasks.t
+TEST_PG='DBI:Pg:dbname=openqa_test;host=/dev/shm/tpg' OPENQA_BASEDIR= prove -Iexternal/os-autoinst-common/lib -v t/14-grutasks.t
```

Comment:
I would really appreciate if we can find a better approach. I do not want to call prove with this additional argument always and also I do not want to tell our developers everytime what's wrong

Reply:
Maybe we can add a symlink?
Or add a `use lib ...` in every test, like we already do with `use lib "$FindBin::Bin/lib";`